src/components/select/select.tsx

```diff
@@ -255,11 +255,28 @@ export class Select {
     ) {
         event.stopPropagation();
 
+        const selector = `#${this.portalId} limel-menu-surface limel-list`;
```

Comment:
```suggestion
        const selector = `#${this.portalId} limel-menu-surface limel-list ul`;
```
I believe it's the actual ul component we should be getting the scroll position for.